es-app/src/SystemData.cpp

```diff
@@ -60,7 +60,7 @@ void SystemData::setIsGameSystemStatus()
 	// we exclude non-game systems from specific operations (i.e. the "RetroPie" system, at least)
 	// if/when there are more in the future, maybe this can be a more complex method, with a proper list
 	// but for now a simple string comparison is more performant
-	mIsGameSystem = (mName != "retropie");
+	mIsGameSystem = (mName != "retropie") && (mName != "kodi");
```

Comment:
Thanks. I'm definitely open to removing that.

The reason I added is that I noticed I have it as a system, but only in retrospect did I find that it's not a default, but rather how I set it up based on the recommendation in the wiki:

https://github.com/RetroPie/RetroPie-Setup/wiki/KODI#kodi-as-its-own-system-instead-of-in-ports

Every now and then a few users in the forums comment that their es_systems.cfg isn't being updated when a new system is installed, and it turns out it's because they followed that guideline, so I thought I'd include it as a semi-common case.

But you are correct, even though a "Kodi" system is not meant to be a game system (and that's what we are trying to limit the Random games to), it doesn't need to be here as it shouldn't be a system in the first place on a clean RetroPie setup.

I won't object to removing it - happy to hear your thoughts!

Best.